Content.Server/Actions/Actions/GhostBoo.cs

```diff
@@ -31,12 +38,11 @@ public void DoInstantAction(InstantActionEventArgs args)
             var booCounter = 0;
             foreach (var ent in ents)
             {
-                var boos = ent.GetAllComponents<IGhostBooAffected>().ToList();
-                foreach (var boo in boos)
-                {
-                    if (boo.AffectedByGhostBoo(args))
-                        booCounter++;
-                }
+                var ghostBoo = new GhostBooEvent();
+                EntityManager.EventBus.RaiseLocalEvent(ent.Uid, ghostBoo);
```

Comment:
you can get a ref to EntityManager by just doing ent.EntityManager, no need to DI